### PR TITLE
Feature/omega 163 poweroutage map legend

### PR DESCRIPTION
**Pull request by erlbe**
OMEGA-163

## What is the reason for this change?

Display and explain the colours of the poweroutage map.

### What is the expected result?

Displaying a legend which shows what the orange and red colour means.

Desired result displayed in Figma: https://www.figma.com/file/sJHoScmQRq0hlr6AcHI9WB/Elvia.no?node-id=51%3A1

## How is it solved?

Add a Legend component to Poweroutage package. Uses termContext to get the terms to display.
image

## Any changes/updates that may affect others?

|                       | Changed |
| --------------------- | ------- |
| Unit tests            |         |
| Dependencies          |         |
| Environment variables |         |
| Build scripts         |         |

## How can it be tested and verified?

- [ ] /strombruddskartv2 should display the legend. However not placed on the map yet.

### CODE REVIEW

File: packages/poweroutage/src/components/Legend/Legend.module.scss [hunk 1]
```diff
@@ -0,0 +1,42 @@
+@import '../../../../ui/src/theme';
```
Comment (t0ff3r) [inline packages/poweroutage/src/components/Legend/Legend.module.scss:1]: This should probably be imported in some other way?

File: packages/poweroutage/src/index.tsx [hunk 2]
```diff
@@ -108,6 +109,9 @@ export const PowerOutage: React.FC<PowerOutageProps> = ({ terms }) => (
           {table}
         </InfoBox>
       </div>
+      <div style={{ width: '336px' }}>
```
Comment (t0ff3r) [inline packages/poweroutage/src/index.tsx:112]: Should this be in a class instead?

File: packages/poweroutage/src/components/Legend/Legend.module.scss (hunk 1 above)
Comment (FredricW) [inline packages/poweroutage/src/components/Legend/Legend.module.scss:1]:
```suggestion
@import '@elvia/ui/src/_theme.scss';

Funker =)
```

Comment (erlbe) [reply packages/poweroutage/src/components/Legend/Legend.module.scss:1]: Da bruker jeg det 👍

File: packages/poweroutage/src/index.tsx (hunk 2 above)
Comment (erlbe) [reply packages/poweroutage/src/index.tsx]: Added css styling in class to set width and position the legend.

**Review by t0ff3r — APPROVED** [review]

**Review by FredricW — APPROVED** [review]

**Pull request merged**